src/plugins/gui/scenes/FiatPluginEnterAmountScene.tsx

```diff
@@ -187,133 +186,126 @@ export const FiatPluginEnterAmountScene = React.memo((props: Props) => {
+                    verticalRem={0.5}
+                  />
+                </EdgeAnim>
+                {onMax != null ? (
+                  <View style={styles.maxButton}>
+                    <ButtonsView
+                      tertiary={{
+                        label: lstrings.string_max_cap,
+                        onPress: handleMax
+                      }}
+                    />
+                  </View>
+                ) : null}
+              </View>
+            )}
+            <>
+              <EdgeAnim enter={fadeInDown30}>
+                <EdgeText numberOfLines={2} style={statusTextStyle}>
+                  {statusText.content}
+                </EdgeText>
               </EdgeAnim>
-              <EdgeAnim enter={fadeInUp30}>
-                <FilledTextInput
-                  disabled={disableInput === 2}
-                  numeric
-                  maxDecimals={6}
-                  autoCorrect={false}
-                  autoFocus={false}
-                  autoCapitalize="none"
-                  keyboardType="decimal-pad"
-                  placeholder={label2}
-                  onChangeText={handleChangeText2}
-                  onSubmitEditing={handleSubmit}
-                  returnKeyType="done"
-                  showSpinner={spinner2}
-                  textsizeRem={1.5}
-                  value={value2 ?? '0'}
-                  verticalRem={0.5}
-                />
+              <EdgeAnim enter={fadeInDown60}>
+                <PoweredByCard iconUri={poweredByIconPath} poweredByText={poweredBy?.poweredByText ?? ''} onPress={handlePoweredByPress} />
               </EdgeAnim>
-              {onMax != null ? (
-                <View style={styles.maxButton}>
-                  <ButtonsView
-                    tertiary={{
-                      label: lstrings.string_max_cap,
-                      onPress: handleMax
-                    }}
-                  />
-                </View>
-              ) : null}
-            </View>
-          )}
-          <>
-            <EdgeAnim enter={fadeInDown30}>
-              <EdgeText numberOfLines={2} style={statusTextStyle}>
-                {statusText.content}
-              </EdgeText>
-            </EdgeAnim>
-            <EdgeAnim enter={fadeInDown60}>
-              <PoweredByCard iconUri={poweredByIconPath} poweredByText={poweredBy?.poweredByText ?? ''} onPress={handlePoweredByPress} />
-            </EdgeAnim>
-            <EdgeAnim enter={fadeInDown90}>
-              <MainButton disabled={spinner1 || spinner2} label={lstrings.string_next_capitalized} marginRem={[0.5, 0]} onPress={handleSubmit} />
-            </EdgeAnim>
-          </>
-        </View>
-      </SectionView>
-    </SceneWrapper>
+            </>
+          </View>
+        </SceneContainer>
+      </SceneWrapper>
+      <KavButton disabled={spinner1 || spinner2} label={lstrings.string_next_capitalized} onPress={handleSubmit} hasTabs hasNotifications />
```

Comment:
In the `KavButton` comment, you said this needs to be child of `SceneWrapper`. Here, it's a sibling. We either need to fix the comment, or we need to fix this code.